.github/actions/setup-dependencies/action.yml

```diff
@@ -0,0 +1,41 @@
+inputs:
+  python-version:
+    description: "Python version to setup"
+    required: false
+    default: "3.8"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v2
```

Comment:
I think this is the only action we need for publish, which can be inlined in `release.yml`

Reply:
I inlined python and poetry installation in `release.yml`